### ISSUE TITLE
Focus ring missing on Catalog/Articles+ search index field

### ISSUE THREAD

**Issue by saseestone** [sul-dlss/SearchWorks#6304]
It's confusing visual users using keyboard shortcuts about where they are, and that they can use arrow keys to view & select from the dropdown.

Reported via feedback as something the user couldn't do at all, and could we restore the ability to use the keyboard to select the index and them move to the search term field. [SW-4576](https://jirasul.stanford.edu/jira/browse/SW-4576)
